addons/skin.estuary/xml/DialogSeekBar.xml

```diff
@@ -223,6 +223,17 @@
 					<visible>Player.SeekEnabled</visible>
 				</control>
 			</control>
+			<control type="ranges" id="404">
```

Comment:
This probably needs a .dox document describing the reasoning and the available tags (see for instance: https://github.com/xbmc/xbmc/blob/6fcbb13a90d8cd9782e4bfaee9ea19459e7aa45d/xbmc/guilib/GUISliderControl.dox). I can do it later if you don't want to be bothered with documentation pings :)

Reply:
your help on this is greatly appreciated